hkmc2/shared/src/main/scala/hkmc2/semantics/Resolver.scala

```diff
@@ -101,7 +101,7 @@ object Resolver:
     
     def describeType(tpe: Type): Str = tpe match
       case Type.Sym(sym: VarSymbol) =>
-        s"${tEnv.get(sym).getOrElse(Type.Unspecified).show} (${tpe.show})"
+        s"${tEnv.get(sym).getOrElse(Type.Unspecified).show} (type variable ${tpe.show})"
       case _ => 
         s"${tpe.show}"
```

Comment:
I changed the error message a bit as I found the previous one was a bit ambiguous